src/scripts/main.js

```diff
@@ -1,3 +1,109 @@
 'use strict';
 
-// write code here
+const TBODY = document.querySelector('.field tbody');
```

Comment:
Code smell: It's not recommended to use global variables. It's better to pass `TBODY` as a parameter to the functions that need it. Global variables can cause problems with naming conflicts and debugging.